README.md

```diff
@@ -57,7 +65,11 @@ Clone the repo and create all shells' final `aws-cli-mfa-*` scripts and then cop
     AWS_CLI_MFA_SHELL=bash &&
     ./build.sh && sudo cp ./bin/aws-cli-mfa-$AWS_CLI_MFA_SHELL /usr/local/bin/aws-cli-mfa
 
-Current shell options are: `bash`, `zsh`, `ksh`
+Or if using Fish:
+
+    ./build.fish
```

Comment:
Does this also copy it to bin, like the above instructions do? If not, please include the copy instructions on the line as well

Reply:
Yes, it is the same command as is in the `build.sh` file.  As per the comment below, they actually all copy it into `/usr/local/bin`, but since the fish script only has the one command we don't need to do the extra `rm` and then copy it back.